README.md

```diff
@@ -1,5 +1,24 @@
-# Todo MVC with ReactJS + redux-saga + Redux
+# Todo MVC with ReactJS + Redux-saga + Redux
 [![CircleCI](https://circleci.com/gh/alakarteio/todomvc.svg?style=svg)](https://circleci.com/gh/alakarteio/todomvc) [![Coverage Status](https://coveralls.io/repos/github/alakarteio/todomvc/badge.svg?branch=master)](https://coveralls.io/github/alakarteio/todomvc?branch=master)
 
-# Demo
-[![todomvc](https://image.noelshack.com/fichiers/2017/44/7/1509875726-todo.png)](https://alakarteio.github.io/todomvc/)
+# Application
+<p align="center">
+  <a href="https://alakarteio.github.io/todomvc/" target="_blank">
+    <img src="https://image.noelshack.com/fichiers/2017/44/7/1509875726-todo.png" />
```

Comment:
this screenshot refer to `Alakarte` not `alakarte`

Reply:
@guillaumecrespel what about this ?

Reply:
My OS not show emoji ... i take screenshot later this evening